src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -37,10 +37,10 @@ public Test()
 
                 SolutionTransforms.Add((solution, projectId) =>
                 {
-                    var parseOptions = (CSharpParseOptions)solution.GetRequiredProject(projectId).ParseOptions!;
+                    var parseOptions = (CSharpParseOptions)solution.GetProject(projectId)?.ParseOptions!;
                     solution = solution.WithProjectParseOptions(projectId, parseOptions.WithLanguageVersion(LanguageVersion));
 
-                    var compilationOptions = solution.GetRequiredProject(projectId).CompilationOptions!;
+                    var compilationOptions = solution.GetProject(projectId)?.CompilationOptions!;
```

Comment:
❕ The same applies here. `GetRequiredProject` is the desired behavior, but this would be a fallback if it cannot be used:

```suggestion
                    var compilationOptions = solution.GetProject(projectId)!.CompilationOptions!;
```